### PR TITLE
Add missing German translations

### PR DESCRIPTION
**Pull request by der-lyse**

### CODE REVIEW

File: po/de.po [hunk 1]
```diff
@@ -148,34 +146,31 @@ msgstr ""
...
 
 #: newsboat.cpp:167
-#, fuzzy
 msgid ""
 "- optional-lite library, licensed under the Boost Software License: https://"
 "github.com/martinmoene/optional-lite"
 msgstr ""
-"Es wird die unter der MIT-Lizenz stehende Bibliothek JSON für Modernes C++ "
-"verwendet: https://github.com/nlohmann/json"
+"- optional-lite, unter der MIT-Lizenz stehende Bibliothek: https://"
```
Comment (Minoru) [inline po/de.po:164]: This mentions MIT, but the license is actually BSL. Ditto with expected-lite, below.

File: po/de.po [hunk 2]
```diff
@@ -1494,9 +1493,8 @@ msgid "`%s' is not a valid key command"
 msgstr "`%s' ist kein gültiges Tastenkommando"
 
 #: src/keymap.cpp:744
-#, fuzzy, c-format
```
Comment (Minoru) [inline po/de.po:1497]: I think `c-format` should've been kept, since the string contains `%s`.

File: po/de.po (hunk 1 above)
Comment (der-lyse) [reply po/de.po:164]: Oh, indeed! I must have gotten confused by the fuzzy translation.

File: po/de.po (hunk 2 above)
Comment (der-lyse) [reply po/de.po:1497]: You're right, I just `dd`ed without paying attention.

**Pull request merged**